dev/io.openliberty.microprofile.metrics.5.0.internal/src/io/openliberty/smallrye/metrics/cdi/adapters/MetricRegistryProducerAdapter.java

```diff
@@ -4,7 +4,7 @@
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-2.0/
- * 
```

Comment:
Update Copyright year

Reply:
file has spacing changes, copyright not needed